lib/kb_Metrics/metricsDBs.py

```diff
@@ -34,12 +34,16 @@ def __init__(self, mongo_host, mongo_dbs, mongo_user, mongo_psswd):
         self.metricsDBs = dict()
         for m_db in mongo_dbs:
             # create the client and authenticate
-            self.mongo_clients[m_db] = MongoClient(
-                "mongodb://" + mongo_user + ":" + mongo_psswd +
-                "@" + mongo_host + "/" + m_db)
+            if mongo_host:
+                self.mongo_clients[m_db] = MongoClient(
+                    "mongodb://" + mongo_user + ":" + mongo_psswd +
```

Comment:
In metricsdb_controller mongo_user and mongo_psswd is optional. It breaks this. So either fix metricsdb_controller.__init__() or add check here.

Reply:
I will add check on this.

Reply:
I added the check in  metricsdb_controller to enforce the check.